packages/twenty-front/src/modules/activities/components/ExpandableList/IntersectionObserverWrapper.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useInView } from 'react-intersection-observer';
+
+export const IntersectionObserverWrapper = ({
```

Comment:
I'm a bit worried about performances here to be honnest. Having one on the page is fined. Having many will impact the performance. I don't see a workaround. It looks like we need:
- this ExpandableList component or the one that has been forced to 2 chips for ActivityTarget
- an expandableList for table view that do not care about the intersection observer and is hiding the overflow with CSS without computing anything

I would love to discuss this with Design

Reply:
@Bonapara